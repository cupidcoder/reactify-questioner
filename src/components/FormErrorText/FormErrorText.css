.FormErrorText {
  font-size: 0.8rem;
  color: rgb(221, 130, 130);
  text-align: center;
}
