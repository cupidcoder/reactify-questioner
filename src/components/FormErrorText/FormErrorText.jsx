import React from 'react';
import style from './FormErrorText.css';

const FormErrorText = ({ error }) => (
  <p className={style.FormErrorText}>{error}</p>
);

export default FormErrorText;
