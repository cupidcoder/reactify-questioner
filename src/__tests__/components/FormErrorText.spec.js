import React from 'react';
import { shallow } from 'enzyme';
import FormErrorText from '../../components/FormErrorText/FormErrorText';

describe('FormErrorText', () => {
  it('Should correctly save snapshot of the FormErrorText', () => {
    const FormErrorTextWrapper = shallow(<FormErrorText />);
    expect(FormErrorTextWrapper).toMatchSnapshot();
  });
});
