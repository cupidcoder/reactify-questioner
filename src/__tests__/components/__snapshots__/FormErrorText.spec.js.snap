// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormErrorText Should correctly save snapshot of the FormErrorText 1`] = `ShallowWrapper {}`;
